ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/HikariJdbcConnectionProvider.scala

```diff
@@ -43,10 +64,16 @@ object HikariConnection {
       config.setPoolName(s"$ConnectionPoolPrefix.${serverRole.threadPoolSuffix}")
       metrics.foreach(config.setMetricRegistry)
 
-      //note that Hikari uses auto-commit by default.
-      //in `runSql` below, the `.close()` will automatically trigger a commit.
-      new HikariDataSource(config)
-    })
+      // Hikari dies if a database connection could not be opened almost immediately
+      // regardless of any connection timeout settings. We retry connections so that
+      // Postgres and Sandbox can be started in any order.
+      Resource(
+        RetryStrategy.constant(attempts = 600, waitTime = 1.second) { (_, _) =>
+          Future { new HikariDataSource(config) }
```

Comment:
I would add a log line before this one to give some feedback as to what's going on. The parameters of the anonymous function should give some meaningful information. `info` level should be enough.